course_contents/2_intro_to_python/notes/04. Destructuring.md

```diff
@@ -17,7 +15,7 @@ print(a, b, c)
 ```
 
 ### The `enumerate()` method
-- the `enumerate()` method is used to retrieve an object containing both element and its index inside the collection that is being iterated over
+- the `enumerate()` method is used to retrieve an object containing both element and its index inside the itereable that is being iterated over
```

Comment:
I'd bring back `collection` if we have decided (from the top of this PR) that we're not going to talk about `iterable`.